Formula/x/xml2rfc.rb

```diff
@@ -0,0 +1,151 @@
+class Xml2rfc < Formula
+  include Language::Python::Virtualenv
+
+  desc "Tool to convert XML RFC7749 to the original ASCII or the new HTML look-and-feel"
+  homepage "https://xml2rfc.tools.ietf.org/"
+  url "https://files.pythonhosted.org/packages/19/5a/4e4cbd0d669ff6bd368ba185bfbad3fb91ac4d500374398d2e245ddc5394/xml2rfc-3.28.1.tar.gz"
+  sha256 "76201b28f5b5d4c466dad74d972b3260013db0f462c76ebc633ea1c7fcd87194"
+  license "BSD-3-Clause"
+  head "https://github.com/ietf-tools/xml2rfc.git", branch: "main"
+
+  depends_on "python@3.12"
```

Comment:
does it build with py3.13?

```suggestion
  depends_on "python@3.13"
```